lib/interop.js

```diff
@@ -468,7 +468,6 @@ Interop.prototype.toUnifiedPlan = function(desc) {
                         // msid. Note that "recvonly" is a term only defined
                         // for m-lines.
                         uLine.msid = sources[ssrc].msid;
-                        uLine.direction = 'sendrecv';
```

Comment:
You're right, sdp-interop is a translator, but with a few quirks because Plan B offer/answer doesn't have enough information to create a Unified Plan offer/answer.

In this particular case, we're using a Plan B line as a prototype to create all the Unified Plan lines that are missing. TBH I don't remember the exact situation that was failing (hence the need for tests!) but if the Plan B line is recvonly, then all the created unified lines are recvonly and this  caused a problem.

Does this break any of your tests? Would it be possible to move the tests in a separate PR and then discuss about this one?